server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -4,7 +4,7 @@
 const LogicalTerminationPoint = require('onf-core-model-ap/applicationPattern/onfModel/models/LogicalTerminationPoint');
 const LogicalTerminationPointConfigurationInput = require('onf-core-model-ap/applicationPattern/onfModel/services/models/logicalTerminationPoint/ConfigurationInput');
```

Comment:
I think this shall also be updated as ConfigurationInputV2.js and corresponding implementation could be updated accordingly.

Reply:
Thank you, removed the ConfigurationInput declaration as in the latest we are not using that module.